bindings/nodejs/tests/binding.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const assert = require('assert')
+const { Operator } = require('../index.js')
+const { Given, When, Then, } = require('@cucumber/cucumber')
+
+Given('A new OpenDAL Blocking Operator', () => {
```

Comment:
We should use anonymous function form `function(){}.` An arrow function will get the wrong `this`, which means  the context. Including all the steps.
```javascript
Given('A new OpenDAL Blocking Operator', function() {
    this.op = new Operator("memory")
});
```

Reply:
DONE